runner/src/server/plugins/engine/components/MultiInputField.ts

```diff
@@ -65,9 +65,13 @@ export class MultiInputField extends FormComponent {
     const stringValue = new Array();
     if (values) {
       for (var value of values) {
-        stringValue.push(
-          `${value["type-of-revenue-cost"]} : ${this.options.prefix}${value["value"]}`
-        );
+        if (typeof value === "string") {
+          stringValue.push(value);
+        } else {
+          stringValue.push(
+            `${value["type-of-revenue-cost"]} : ${this.options.prefix}${value["value"]}`
```

Comment:
Is this re-usable if we assume value is always `type-of-revenue-cost`?

Reply:
It will always be type-of-revenue-cost due to how multi input components work
![image](https://user-images.githubusercontent.com/97108643/195378001-0dcf6b9b-6368-4a9e-873f-89a8b8f9762e.png)

Unless someone changes it in code as well, I think we have a tech debt ticket for reworking MultiInput Fields